tests/Entity/BaseEntityManagerTest.php

```diff
@@ -16,74 +16,99 @@
 use Doctrine\Common\Persistence\ManagerRegistry;
 use Doctrine\Common\Persistence\ObjectManager;
 use Doctrine\ORM\EntityRepository;
+use PHPUnit\Framework\MockObject\MockObject;
 use PHPUnit\Framework\TestCase;
 use Sonata\Doctrine\Entity\BaseEntityManager;
 
-class EntityManager extends BaseEntityManager
-{
-    public function getRepositoryFromBaseClass(): EntityRepository
-    {
-        return $this->getRepository();
-    }
-}
-
+/**
+ * Class BaseEntityManagerTest.
+ */
 final class BaseEntityManagerTest extends TestCase
 {
-    public function getManager()
+    /**
+     * @var ManagerRegistry|MockObject
+     */
+    private $registry;
+
+    /**
+     * @var ObjectManager|MockObject
+     */
+    private $objectManager;
+
+    /**
+     * @var BaseEntityManager
+     */
+    private $manager;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setUp(): void
     {
-        $registry = $this->createMock(ManagerRegistry::class);
-
-        return new EntityManager('classname', $registry);
+        $this->registry = $this->createMock(ManagerRegistry::class);
+        $this->objectManager = $this->createMock(ObjectManager::class);
+        $this->manager = $this->getMockForAbstractClass(BaseEntityManager::class, ['classname', $this->registry]);
     }
 
-    public function test()
+    /**
+     * Test get class name.
+     */
```

Comment:
If you think the method name is bad, feel free to change it but please don't add this comment. Same for the methods below please.